apps/web/src/pages/liquid-staking/[contract].tsx

```diff
@@ -113,7 +113,7 @@ const LiquidStakingStakePage = () => {
                 tokenAddress={selectedList?.token1?.address}
                 tokenSymbol={outputCurrency?.symbol}
                 tokenDecimals={outputCurrency?.decimals}
-                tokenLogo={undefined}
+                tokenLogo={`${window?.location?.origin}/images/tokens/${selectedList?.token1?.address}.png`}
```

Comment:
accessing `window` on SSR is not quite safe. Can we build this url on user interaction, which can help make sure we're on the client side? Or at least add a typeof check for `window`?

Reply:
although I know for this component right now we're not rendering anything on server side